exe/kubernetes-deploy

```diff
@@ -43,16 +45,22 @@ revision = ENV.fetch('REVISION') do
   exit 1
 end
 
-KubernetesDeploy::Runner.with_friendly_errors do
-  runner = KubernetesDeploy::Runner.new(
-    namespace: ARGV[0],
-    context: ARGV[1],
-    current_sha: revision,
-    template_dir: template_dir,
-    wait_for_completion: !skip_wait,
-    allow_protected_ns: allow_protected_ns,
-    prune: prune,
-    bindings: bindings
-  )
-  runner.run
-end
+namespace = ARGV[0]
+context = ARGV[1]
+logger = KubernetesDeploy::FormattedLogger.build(namespace, context, verbose_prefix: verbose_log_prefix)
+
+runner = KubernetesDeploy::Runner.new(
+  namespace: namespace,
+  context: context,
+  current_sha: revision,
+  template_dir: template_dir,
+  bindings: bindings,
+  logger: logger
+)
+
+success = runner.run(
+  verify_result: !skip_wait,
```

Comment:
What was the reason to change `wait_for_completion` to `verify_result`? What this flag enables is waiting for resources to become available. I think it's still more _waiting_ than _verifying_